official/resnet/ctl/ctl_imagenet_main.py

```diff
@@ -215,70 +228,44 @@ def step_fn(inputs):
       else:
         step_fn(test_ds_inputs)
 
-    if flags_obj.enable_function:
+    if flags_obj.use_tf_function:
       train_step = tf.function(train_step)
       test_step = tf.function(test_step)
 
-    epoch_exp_per_sec = []
-    stats = {}
+    time_callback.on_train_begin()
     for epoch in range(train_epochs):
-      logging.info('Starting to run epoch: %s', epoch)
       train_iterator = iter(train_ds)
 
       step = 0
```

Comment:
i am confused why there seem to be 2 things called `step` here..

Reply:
I think the first one was added when I was iterating over the distributed dataset. I removed that version since there were a few outstanding issues at that time.